python/src/experimental_module.i.orig

```diff
@@ -0,0 +1,71 @@
+// SWIG file experimental_module.i
```

Comment:
This looks that a git merge file problem here: this file should probably not be here.